cl/disclosures/tests.py

```diff
@@ -55,6 +63,29 @@ def test_financial_disclosure_ingestion(self):
             % non_investments.count(),
         )
 
+    def test_extraction_and_ingestion_jef(self):
+        """Can we successfully ingest disclosures from jef documents?"""
+        with open(self.jef_pdf, "rb") as f:
+            pdf_bytes = f.read()
+        Investment.objects.all().delete()
+        extractor_response = requests.post(
+            settings.BTE_URLS["extract-disclosure-jef"]["url"],
+            files={"file": ("file", pdf_bytes)},
+            timeout=settings.BTE_URLS["extract-disclosure-jef"]["timeout"],
+        )
+        extracted_data = extractor_response.json()
+        test_disclosure = FinancialDisclosure.objects.get(pk=1)
+        save_disclosure(
+            extracted_data=extracted_data,
+            disclosure=test_disclosure,
+        )
+        investments = Investment.objects.all()
+        self.assertTrue(
+            investments.count() == 84,
+            "Should have 19 ingested investments, not %s"
+            % investments.count(),
```

Comment:
This should use `assertEqual`, it'll give you the nice error message itself. Also it seems like you're expecting 84 items, but then the message says 19? Also, you should avoid doing the count twice.